internal/pkg/scorecard/helpers.go

```diff
@@ -35,3 +40,85 @@ func ResultsCumulative(results []TestResult) (earned, max int) {
 	}
 	return earned, max
 }
+
+// CalculateResult returns a ScorecardSuiteResult with the state and Tests fields set based on a slice of ScorecardTestResults
+func CalculateResult(tests []*scapiv1alpha1.ScorecardTestResult) *scapiv1alpha1.ScorecardSuiteResult {
+	scorecardSuiteResult := scapiv1alpha1.ScorecardSuiteResult{}
+	scorecardSuiteResult.Tests = tests
+	for _, test := range scorecardSuiteResult.Tests {
+		scorecardSuiteResult.TotalTests++
+		switch test.State {
+		case scapiv1alpha1.ErrorState:
+			scorecardSuiteResult.Error++
+		case scapiv1alpha1.PassState:
+			scorecardSuiteResult.Pass++
+		case scapiv1alpha1.PartialPassState:
+			scorecardSuiteResult.PartialPass++
+		case scapiv1alpha1.FailState:
+			scorecardSuiteResult.Fail++
+		}
+	}
+	return &scorecardSuiteResult
+}
+
+// TestSuitesToScorecardOutput takes an array of test suites and generates a v1alpha1 ScorecardOutput object with the
+// provided name, description, and log
+func TestSuitesToScorecardOutput(suites []*TestSuite, log string) *scapiv1alpha1.ScorecardOutput {
+	test := &scapiv1alpha1.ScorecardOutput{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ScorecardOutput",
+			APIVersion: "osdk.scorecard.com/v1alpha1",
```

Comment:
Should this be `"osdk.openshift.io/v1alpha1"`?

Reply:
Yup.